### PR TITLE
Update foreman-tasks to 0.6.14

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: plugins/ruby-foreman-tasks/debian/control [hunk 1]
```diff
@@ -2,12 +2,12 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.7)
+Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.9)
```
Comment (domcleal) [inline plugins/ruby-foreman-tasks/debian/control]: and also < 0.8.0

File: plugins/ruby-foreman-tasks/debian/control [hunk 2]
```diff
@@ -2,12 +2,12 @@ Source: ruby-foreman-tasks
...
 Standards-Version: 3.9.3
 Homepage: https://github.com/Dynflow/dynflow
 
 Package: ruby-foreman-tasks
 Architecture: all
-Depends: ${misc:Depends}, bundler, foreman (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.7)
+Depends: ${misc:Depends}, bundler, foreman (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.9)
```
Comment (domcleal) [inline plugins/ruby-foreman-tasks/debian/control]: ditto

File: plugins/ruby-foreman-tasks/debian/control (hunk 1 above)
Comment (iNecas) [inline plugins/ruby-foreman-tasks/debian/control]: updated (I hope it works this way)

**Comment by iNecas** [comment]
Jenkins is failing: what am I doing wrong?

**Comment by domcleal** [comment]
Seems our nightly repo only contains 0.7.8 and not 0.7.9 from #683, so I'm just going to try building that, maybe it didn't happen or failed after the PR.  This PR must have gone green the first time as it was using a scratch build from your repo (version '9999', which the pinning then prevented).

**Comment by domcleal** [comment]
[test]

**Comment by domcleal** [comment]
Thanks @iNecas!

**Pull request merged**